dapp/src/components/buySell/SwapHomepage.js

```diff
@@ -516,7 +592,7 @@ const SwapHomepage = ({
             }}
           />
         )}
-        {showApproveModal && (
+        {/* {showApproveModal && (
```

Comment:
cleanup if it is dead code. same comment for other commented out lines of code below.